src/consensus/aft/raft.h

```diff
@@ -430,6 +469,11 @@ namespace aft
 
     void add_configuration(Index idx, const Configuration::Nodes& conf)
     {
+      for (auto& [id, info] : conf)
+      {
+        LOG_TRACE_FMT(
+          "CADD {} {}:{} {}", id, info.hostname, info.port, info.catching_up);
+      }
```

Comment:
I think the logs are more readable if we're more verbose:

```suggestion
      LOG_TRACE_FMT("Adding a configuration at {} of {} nodes", idx,  conf.size());
      for (auto& [id, info] : conf)
      {
        LOG_TRACE_FMT(
          "  {} = {}:{} ({})", id, info.hostname, info.port, info.catching_up ? "passive" : "active");
      }
```